app/src/main/java/com/grzegorzdec/tagmango/common/recyclerview/RecyclerViewBindingUtils.kt

```diff
@@ -0,0 +1,18 @@
+package com.grzegorzdec.tagmango.common.recyclerview
+
+import androidx.databinding.BindingAdapter
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.grzegorzdec.tagmango.seller.map.ClientsRecyclerAdapter
+
+@BindingAdapter("listBinder")
+fun <E> bindItems(recyclerView: RecyclerView, listBinder: ListBinder<E>) {
```

Comment:
just an idea, you can create `fun <E> RecyclerView.bindItems(listBinder: ListBinder<E>) {`